### PR TITLE
add smi paramters for NY

### LINKED ISSUE #280: Add SMI parameters for NY
State median income by state and family size

https://www.acf.hhs.gov/sites/default/files/documents/ocs/COMM_LIHEAP_IM03%20Attachment1%20SMITable_FY2022.pdf

### PR DESCRIPTION
**Pull request by tolaouk**
Fixes #280

### CODE REVIEW

File: openfisca_us/parameters/computed/smi.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+description: State median income by state and family size
```
Comment (MaxGhenis) [inline openfisca_us/parameters/computed/smi.yaml:1]:
```suggestion
description: State median income for 4-person households by state
```

File: openfisca_us/parameters/computed/smi.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+NY:
+  4:
```
Comment (MaxGhenis) [inline openfisca_us/parameters/computed/smi.yaml:4]: Remove this layer

**Review by MaxGhenis — COMMENTED** [review]
Per #276 could you put this in `parameters/hhs/smi.yaml` instead?

File: openfisca_us/parameters/benefit/hhs/smi_one_person.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+description: State median income for 1-person households by state
```
Comment (MaxGhenis) [inline openfisca_us/parameters/benefit/hhs/smi_one_person.yaml:1]: This is unnecessary - HHS applies a constant adjustment factor across all states: https://github.com/PolicyEngine/openfisca-us/issues/263

e.g. this is 52% of the 4-person. So we only need the 4-person.

File: openfisca_us/parameters/benefit/hhs/smi.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+description: State median income for 4-person households by state
+
+NY:
+  2021-10-01: 104_972
+
+
```
Comment (MaxGhenis) [inline openfisca_us/parameters/benefit/hhs/smi.yaml:6]: remove spare line

**Review by MaxGhenis — COMMENTED** [review]
Good to merge after this and removing `benefit/` from the path - we're now just going with `parameters/[agency]`

**Comment by tolaouk** [comment]
I probably should've moved the ccdf county cluster parameter to hhs folder under a different PR. Hope this is okay.

**Pull request merged**